### PR TITLE
add python-albumentations-pip

### PR DESCRIPTION
**Pull request by fmessmer**
Please add the following dependency to the rosdep database.

## Package name:

`python-albumentations-pip`

## Package Upstream Source:
https://pypi.org/project/albumentations/
(pip only package - no release found, see below)

## Purpose of using this:

https://pypi.org/project/albumentations/#description

Distro packaging links:

## Links to Distribution Packages

- Debian: https://packages.debian.org/
  - NOT_FOUND
- Ubuntu: https://packages.ubuntu.com/
   - NOT_FOUND
- Fedora: https://src.fedoraproject.org/browse/projects/
  - NOT_FOUND
- Arch: https://www.archlinux.org/packages/
  - NOT_FOUND
- Gentoo: https://packages.gentoo.org/
  - NOT_FOUND
- macOS: https://formulae.brew.sh/
  - NOT_FOUND
- Alpine: https://pkgs.alpinelinux.org/packages
  - NOT_FOUND

### CODE REVIEW

**Comment by fmessmer** [comment]
friendly ping :wink: 
anything wrong with this PR? anything blocking the merge of this PR?

**Comment by fmessmer** [comment]
@clalancette 
are `rosdep` prs not merged anymore/atm?

File: rosdep/python.yaml [hunk 1]
```diff
@@ -434,6 +434,19 @@ python-adafruit-bno055-pip:
   ubuntu:
     pip:
       packages: [adafruit_bno055]
+python-albumentations-pip:
```
Comment (clalancette) [inline rosdep/python.yaml:437]:
```suggestion
python3-albumentations-pip:
```

**Review by clalancette — CHANGES_REQUESTED** [review]

File: rosdep/python.yaml (hunk 1 above)
Comment (fmessmer) [reply rosdep/python.yaml:437]: is this due to `kinetic` no longer being supported? so any `rosdep` key in the `python.yaml` is now being considered `python3`?

Comment (clalancette) [reply rosdep/python.yaml]: > so any `rosdep` key in the `python.yaml` is now being considered `python3`?

It's a compromise amongst a number of bad options.  Essentially we always prefer the distribution/system key, and those are always named `python3-<something>`.  In order to make the pip key live "near" where the distribution key would live, we are naming all pip keys `python3-<something>-pip`.  This also represents the fact that python2 is no longer really supported, and will completely go out of support once the current host of LTS distributions goes EOL (probably ~2023, when Bionic goes EOL).

**Review by clalancette — APPROVED** [review]

**Review by jacobperron — APPROVED** [review]

**Pull request merged**